src/main/java/de/tum/cit/aet/artemis/communication/web/FaqResource.java

```diff
@@ -115,6 +115,18 @@ public ResponseEntity<FaqDTO> updateFaq(@RequestBody Faq faq, @PathVariable Long
         return ResponseEntity.ok().body(dto);
     }
 
+    /**
+     * @param faq      the faq to be checked *
+     * @param courseId the id of the course the faq belongs to
+     *                     This method throws an exception if a non-instructor in the course tries to set the state of an FAQ to ACCEPTED
+     */
+    private void isAtLeastInstructor(Faq faq, Long courseId) {
```

Comment:
Should tutors be allowed to change an accepted faq to unaccepted again? Otherwise you probably have to verify this here as well

Reply:
Good catch, thank you. I adapted the update method